Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -224,7 +224,7 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::DoPartialTextureUpdates(Tex
 
 	TexCache::iterator iter = textures_by_address.lower_bound(entry_to_update->addr);
 	TexCache::iterator iterend = textures_by_address.upper_bound(entry_to_update->addr + entry_to_update->size_in_bytes);
-	bool entry_need_scaling = true;
+	bool entry_need_scaling = g_ActiveConfig.bCopyEFBScaled;
```

Comment:
You have a point. But i think it's ok for users to get issues, when changing the IR while the game is running. There would be at least small issues anyways, if those textures get rescaled over and over again.